modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/DefaultRenderer.tsx

```diff
@@ -92,6 +92,14 @@ const DefaultRenderer: React.FC<{
 		return <Wrapper options={options}>{value.label}</Wrapper>;
 	}
 
+	if (
+		typeof value === 'object' &&
+		Array.isArray(value) &&
+		typeof value[0] !== 'object'
```

Comment:
What if `value[0]` is a `function`? :trollface:

Reply:
🫤

Reply:
Should we check for `function` also?
I know that it is kind of _soft_ validation. The second item could be anything and will break anyway